packages/widget_toolkit_qr/lib/src/base/theme/qr_scanner_theme.dart

```diff
@@ -4,351 +4,398 @@ import 'package:widget_toolkit/widget_toolkit.dart';
-    EdgeInsets.only(top: WidgetToolkitDesignSystem.light().spacings.xs1)
-  ];
-
-  static List<EdgeInsets> backButtonMarginEdgeInsets = [
-    EdgeInsets.only(
-      left: WidgetToolkitDesignSystem.light().spacings.xs,
-    ),
-    EdgeInsets.only(
-      left: WidgetToolkitDesignSystem.dark().spacings.xs,
-    ),
-  ];
-
-  static List<EdgeInsets> backButtonPaddingEdgeInsets = [
-    EdgeInsets.only(
-      left: WidgetToolkitDesignSystem.light().spacings.xs,
-    ),
-    EdgeInsets.only(
-      left: WidgetToolkitDesignSystem.dark().spacings.xs,
-    ),
-  ];
-
-  static List<EdgeInsets> backButtonContainerPaddingEdgeInsets = [
-    EdgeInsets.all(WidgetToolkitDesignSystem.light().spacings.xss1),
-    EdgeInsets.all(WidgetToolkitDesignSystem.dark().spacings.xss1),
-  ];
-
-  /// endregion
-  /// region spacings
-
-  static List<double> mockQrViewSpacing = [
-    WidgetToolkitDesignSystem.light().spacings.m,
-    WidgetToolkitDesignSystem.dark().spacings.m,
-  ];
-
-  static List<double> qrScannerPageXS = [
-    WidgetToolkitDesignSystem.light().spacings.xs,
-    WidgetToolkitDesignSystem.dark().spacings.xs,
-  ];
-
-  static List<double> linearProgressIndicatorXS = [
-    WidgetToolkitDesignSystem.light().spacings.xs,
-    WidgetToolkitDesignSystem.dark().spacings.xs,
-  ];
-
-  static List<double> qrScannerPageM = [
-    WidgetToolkitDesignSystem.light().spacings.m,
-    WidgetToolkitDesignSystem.dark().spacings.m,
-  ];
-
-  static List<double> backButtonSpacingL = [
-    WidgetToolkitDesignSystem.light().spacings.l,
-    WidgetToolkitDesignSystem.dark().spacings.l,
-  ];
-
-  static List<double> appBarSpacingM = [
-    WidgetToolkitDesignSystem.light().spacings.m,
-    WidgetToolkitDesignSystem.dark().spacings.m,
-  ];
-
-  static List<double> appBarSpacingXXXXL = [
-    WidgetToolkitDesignSystem.light().spacings.xxxxl,
-    WidgetToolkitDesignSystem.dark().spacings.xxxxl,
-  ];
-
-  static List<double> backButtonSpacingXXXL = [
-    WidgetToolkitDesignSystem.light().spacings.xxxl,
-    WidgetToolkitDesignSystem.dark().spacings.xxxl,
-  ];
-
-  static List<double> backButtonSpacingXSS1 = [
-    WidgetToolkitDesignSystem.light().spacings.xss1,
-    WidgetToolkitDesignSystem.dark().spacings.xss1,
-  ];
-
-  static List<double> backButtonWidthSpacingXXXL = [
-    WidgetToolkitDesignSystem.light().spacings.xxxl,
-    WidgetToolkitDesignSystem.dark().spacings.xxxl,
-  ];
-
-  static List<double> backButtonHeightSpacingXXXL = [
-    WidgetToolkitDesignSystem.light().spacings.xxxl,
-    WidgetToolkitDesignSystem.dark().spacings.xxxl,
-  ];
+@TailorMixinComponent()
+class QrScannerTheme extends ThemeExtension<QrScannerTheme>
+    with _$QrScannerThemeTailorMixin {
+  const QrScannerTheme({
+    required this.cameraPermission1,
+    required this.cameraPermission2,
+    required this.cameraPermission3,
+    required this.cameraPermission4,
+    required this.cameraPermission5,
+    required this.cameraPermission6,
```

Comment:
`cameraPermission6` ? what does this mean. Please use more domain specific names